bn_mp_pack.c

```diff
@@ -55,11 +65,12 @@ mp_err mp_export(void *rop, size_t *countp, int order, size_t size,
          if ((err = mp_div_2d(&t, (j == ((size - nail_bytes) - 1u)) ? (int)(8u - odd_nails) : 8, &t, NULL)) != MP_OKAY) {
             goto LBL_ERR;
          }
+
       }
    }
 
    if (countp != NULL) {
-      *countp = count;
+      *countp = count * size;
```

Comment:
No I meant countp should return the number of actually written units. Not the total number of units compute